lms/djangoapps/grades/models.py

```diff
@@ -122,24 +120,6 @@ def from_list(cls, blocks, course_key):
         return cls(blocks, course_key)
 
 
-class VisibleBlocksQuerySet(models.QuerySet):
```

Comment:
**Note to reviewers:** No longer needed once `VisibleBlocks.cached_get_or_create` came in the scene.